backend/modules/publishing/tasks.py

```
@@ -0,0 +1,625 @@
+            try:
+                # Dequeue job from Redis
+                job = redis_manager.rpop(QUEUE_KEY)
+                
+                if job:
+                    # Process job
+                    await process_upload_job(job, db)
+                else:
+                    # No jobs available, wait a bit before checking again
+                    await asyncio.sleep(1)
+                    
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)